packages/xo-web/src/xo-app/home/index.js

```diff
@@ -517,6 +519,18 @@ export default class Home extends Component {
     identity,
   ])
 
+  _getNItemsPerPage = () =>
+    +defined(
+      this.state.homeItemsPerPage,
```

Comment:
Why do you need to also store it in the state? Can't we only rely on the cookies?